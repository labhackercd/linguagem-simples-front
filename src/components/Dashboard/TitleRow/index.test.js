import React from 'react';
import { render, screen } from '@testing-library/react';
import TitleRow from './index';
import {shallow} from "enzyme/build";

it("should render the Title Row section", () => {
    const component = shallow(<TitleRow/>);
    expect(component).toMatchSnapshot();
});
