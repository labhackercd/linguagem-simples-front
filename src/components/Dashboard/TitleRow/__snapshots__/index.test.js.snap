// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Title Row section 1`] = `
<div
  className="makeStyles-body-1"
>
  <WithStyles(ForwardRef(Grid))
    className="makeStyles-titleRow-2"
    container={true}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
    >
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-title-3"
        variant="h3"
      >
        Linha do Tempo 
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      className="makeStyles-buttonContainer-4"
      item={true}
      md={6}
    >
      <WithStyles(ForwardRef(Button))
        className="makeStyles-button-5"
        color="secondary"
        disableElevation={true}
        startIcon={
          <img
            alt="button to init stream"
            src="../../img/init_stream_button_icon.svg"
          />
        }
        variant="contained"
      >
        <h6>
          Iniciar transmiss√£o
        </h6>
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</div>
`;
